### ISSUE TITLE
Supabase start command failing with fatal error "required key API_EXTERNAL_URL missing value"

### ISSUE THREAD

**Issue by pjhartin** [supabase/cli#1382]
Failing to run `supabase start` or `supabase db reset` and receiving the following error message

``` 
{"level":"fatal","msg":"Failed to load configuration: required key API_EXTERNAL_URL missing value","time":"2023-08-17T11:55:10Z"}
```

**To Reproduce**
1. Followed the Local Development docs, installing via `yarn install supabase`
2. Added sample `create_employees_table` migration and `seed.sql` from docs
3. Run `yarn supabase start` 
4. See the error below

```
~/Development/learning via  v18.16.1 took 3s 
❯ yarn supabase start --debug
yarn run v1.22.19
$ /Users/philliphartin/Development/learning/node_modules/.bin/supabase start --debug
Supabase CLI 1.86.1
2023/08/17 21:54:57 Sent Header: Host [api.moby.localhost]
2023/08/17 21:54:57 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:54:57 Send Done
2023/08/17 21:54:57 Recv First Byte
2023/08/17 21:54:57 Sent Header: Host [api.moby.localhost]
2023/08/17 21:54:57 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:54:57 Send Done
2023/08/17 21:55:03 Recv First Byte
2023/08/17 21:55:03 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:03 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:03 Send Done
2023/08/17 21:55:04 Recv First Byte
2023/08/17 21:55:04 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:04 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:04 Send Done
2023/08/17 21:55:04 Recv First Byte
2023/08/17 21:55:04 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:04 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:04 Send Done
2023/08/17 21:55:04 Recv First Byte
2023/08/17 21:55:04 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:04 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:04 Send Done
2023/08/17 21:55:04 Recv First Byte
2023/08/17 21:55:04 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:04 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:04 Send Done
2023/08/17 21:55:04 Recv First Byte
2023/08/17 21:55:04 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:04 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:04 Send Done
2023/08/17 21:55:04 Recv First Byte
2023/08/17 21:55:04 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:04 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:04 Send Done
2023/08/17 21:55:04 Recv First Byte
2023/08/17 21:55:04 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:04 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:04 Send Done
2023/08/17 21:55:04 Recv First Byte
2023/08/17 21:55:04 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:04 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:04 Send Done
2023/08/17 21:55:04 Recv First Byte
2023/08/17 21:55:04 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:04 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:04 Send Done
2023/08/17 21:55:04 Recv First Byte
2023/08/17 21:55:04 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:04 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:04 Send Done
2023/08/17 21:55:04 Recv First Byte
2023/08/17 21:55:04 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:04 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:04 Send Done
2023/08/17 21:55:04 Recv First Byte
2023/08/17 21:55:04 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:04 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:04 Send Done
2023/08/17 21:55:04 Recv First Byte
2023/08/17 21:55:04 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:04 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:04 Send Done
2023/08/17 21:55:04 Recv First Byte
2023/08/17 21:55:04 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:04 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:04 Send Done
2023/08/17 21:55:04 Recv First Byte
2023/08/17 21:55:04 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:04 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:04 Send Done
2023/08/17 21:55:04 Recv First Byte
2023/08/17 21:55:04 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:04 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:04 Send Done
2023/08/17 21:55:04 Recv First Byte
2023/08/17 21:55:04 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:04 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:04 Sent Header: Content-Length [306]
2023/08/17 21:55:04 Sent Header: Content-Type [application/json]
2023/08/17 21:55:04 Send Done
2023/08/17 21:55:04 Recv First Byte
2023/08/17 21:55:04 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:04 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:04 Sent Header: Content-Length [11457]
2023/08/17 21:55:04 Sent Header: Content-Type [application/json]
2023/08/17 21:55:04 Send Done
2023/08/17 21:55:04 Recv First Byte
2023/08/17 21:55:04 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:04 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:04 Sent Header: Content-Length [0]
2023/08/17 21:55:04 Sent Header: Content-Type [text/plain]
2023/08/17 21:55:04 Send Done
2023/08/17 21:55:04 Recv First Byte
2023/08/17 21:55:04 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:04 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:04 Send Done
2023/08/17 21:55:04 Recv First Byte
2023/08/17 21:55:05 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:05 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:05 Send Done
2023/08/17 21:55:05 Recv First Byte
2023/08/17 21:55:06 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:06 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:06 Send Done
2023/08/17 21:55:06 Recv First Byte
2023/08/17 21:55:07 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:07 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:07 Send Done
2023/08/17 21:55:07 Recv First Byte
2023/08/17 21:55:08 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:08 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:08 Send Done
2023/08/17 21:55:08 Recv First Byte
2023/08/17 21:55:09 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:09 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:09 Send Done
2023/08/17 21:55:09 Recv First Byte
2023/08/17 21:55:09 Connect Start: tcp [::1]:54322
2023/08/17 21:55:09 Connect Done: tcp [::1]:54322
2023/08/17 21:55:09 PG Send: {"Type":"StartupMessage","ProtocolVersion":196608,"Parameters":{"database":"postgres","user":"postgres"}}
2023/08/17 21:55:09 PG Recv: {"Type":"AuthenticationSASL","AuthMechanisms":["SCRAM-SHA-256"]}
2023/08/17 21:55:09 PG Send: {"Type":"SASLInitialResponse","AuthMechanism":"SCRAM-SHA-256","Data":"n,,n=,r=zYcd9pwMQcbZszlMZCm9TRWG"}
2023/08/17 21:55:09 PG Recv: {"Type":"AuthenticationSASLContinue","Data":"r=zYcd9pwMQcbZszlMZCm9TRWGlEs2XAli0/eApj+5zMNlnF7Q,s=7/s+YVIlL68cQncu1LgyYw==,i=4096"}
2023/08/17 21:55:09 PG Send: {"Type":"SASLResponse","Data":"c=biws,r=zYcd9pwMQcbZszlMZCm9TRWGlEs2XAli0/eApj+5zMNlnF7Q,p=wwh2KTMcOmVAM/LQL7XGyFmcdvwei/kitCEK7E3TNRc="}
2023/08/17 21:55:09 PG Recv: {"Type":"AuthenticationSASLFinal","Data":"v=rZsXf0lhG0ON5TQaKi8QUZyoO4M32qgqj1GkRY32KdM="}
2023/08/17 21:55:09 PG Recv: {"Type":"AuthenticationOK"}
2023/08/17 21:55:09 PG Recv: {"Type":"ParameterStatus","Name":"application_name","Value":""}
2023/08/17 21:55:09 PG Recv: {"Type":"ParameterStatus","Name":"client_encoding","Value":"UTF8"}
2023/08/17 21:55:09 PG Recv: {"Type":"ParameterStatus","Name":"DateStyle","Value":"ISO, MDY"}
2023/08/17 21:55:09 PG Recv: {"Type":"ParameterStatus","Name":"default_transaction_read_only","Value":"off"}
2023/08/17 21:55:09 PG Recv: {"Type":"ParameterStatus","Name":"in_hot_standby","Value":"off"}
2023/08/17 21:55:09 PG Recv: {"Type":"ParameterStatus","Name":"integer_datetimes","Value":"on"}
2023/08/17 21:55:09 PG Recv: {"Type":"ParameterStatus","Name":"IntervalStyle","Value":"postgres"}
2023/08/17 21:55:09 PG Recv: {"Type":"ParameterStatus","Name":"is_superuser","Value":"off"}
2023/08/17 21:55:09 PG Recv: {"Type":"ParameterStatus","Name":"server_encoding","Value":"UTF8"}
2023/08/17 21:55:09 PG Recv: {"Type":"ParameterStatus","Name":"server_version","Value":"15.1 (Ubuntu 15.1-1.pgdg20.04+1)"}
2023/08/17 21:55:09 PG Recv: {"Type":"ParameterStatus","Name":"session_authorization","Value":"postgres"}
2023/08/17 21:55:09 PG Recv: {"Type":"ParameterStatus","Name":"standard_conforming_strings","Value":"on"}
2023/08/17 21:55:09 PG Recv: {"Type":"ParameterStatus","Name":"TimeZone","Value":"UTC"}
2023/08/17 21:55:09 PG Recv: {"Type":"BackendKeyData","ProcessID":171,"SecretKey":2846228257}
2023/08/17 21:55:09 PG Recv: {"Type":"ReadyForQuery","TxStatus":"I"}
2023/08/17 21:55:09 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:09 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:09 Send Done
2023/08/17 21:55:09 Recv First Byte
2023/08/17 21:55:09 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:09 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:09 Sent Header: Content-Length [306]
2023/08/17 21:55:09 Sent Header: Content-Type [application/json]
2023/08/17 21:55:09 Send Done
2023/08/17 21:55:09 Recv First Byte
2023/08/17 21:55:09 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:09 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:09 Sent Header: Content-Length [2261]
2023/08/17 21:55:09 Sent Header: Content-Type [application/json]
2023/08/17 21:55:09 Send Done
2023/08/17 21:55:09 Recv First Byte
2023/08/17 21:55:09 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:09 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:09 Sent Header: Content-Length [0]
2023/08/17 21:55:09 Sent Header: Content-Type [text/plain]
2023/08/17 21:55:09 Send Done
2023/08/17 21:55:09 Recv First Byte
2023/08/17 21:55:09 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:09 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:09 Send Done
2023/08/17 21:55:09 Recv First Byte
2023/08/17 21:55:09 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:09 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:09 Send Done
2023/08/17 21:55:10 Recv First Byte
2023/08/17 21:55:10 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:10 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:10 Send Done
2023/08/17 21:55:10 Recv First Byte
2023/08/17 21:55:10 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:10 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:10 Sent Header: Content-Length [306]
2023/08/17 21:55:10 Sent Header: Content-Type [application/json]
2023/08/17 21:55:10 Send Done
2023/08/17 21:55:10 Recv First Byte
2023/08/17 21:55:10 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:10 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:10 Sent Header: Content-Length [1889]
2023/08/17 21:55:10 Sent Header: Content-Type [application/json]
2023/08/17 21:55:10 Send Done
2023/08/17 21:55:10 Recv First Byte
2023/08/17 21:55:10 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:10 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:10 Sent Header: Content-Length [0]
2023/08/17 21:55:10 Sent Header: Content-Type [text/plain]
2023/08/17 21:55:10 Send Done
2023/08/17 21:55:10 Recv First Byte
2023/08/17 21:55:10 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:10 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:10 Send Done
2023/08/17 21:55:10 Recv First Byte
{"level":"fatal","msg":"Failed to load configuration: required key API_EXTERNAL_URL missing value","time":"2023-08-17T11:55:10Z"}
2023/08/17 21:55:10 Sent Header: Host [api.moby.localhost]
2023/08/17 21:55:10 Sent Header: User-Agent [Docker-Client/unknown-version (darwin)]
2023/08/17 21:55:10 Send Done
2023/08/17 21:55:10 Recv First Byte
2023/08/17 21:55:10 PG Send: {"Type":"Terminate"}
error running container: exit 1
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Expected behaviour**
Expected successful boot of container with migrations being run and seeder.

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] MacOS on Ventural 13.4.1
 - Version of CLI 1.86.1
 - Version of Node.js 18.16.1
 - Version of Docker Desktop 4.22.0 (117440)
 - Version of Docker Engine 24.0.5

**Additional context**
Found this issue on a fresh install, following the supabase docs on local development - https://supabase.com/docs/guides/cli/local-development

**Comment by movaldivia** [comment]
Me too! D:

**Comment by aimenbouchhimadev** [comment]
I had this issue too after linking the  to the cloud project. 
The linking command creates 2files (project-ref and gotrue-version) inside the supabase/.temp folder
When I deleted the gotrue_version file, the problem is fixed !

**Comment by pjhartin** [comment]
Thank you @aimenbouchhimadev! Your solution worked for me.

**Comment by sweatybridge** [comment]
Closing as duplicate of https://github.com/supabase/cli/issues/1384. We have fixed it in 1.86.2
